mingw.condition_variable.h

```diff
@@ -20,34 +20,24 @@
 
 #ifndef MINGW_CONDITIONAL_VARIABLE_H
 #define MINGW_CONDITIONAL_VARIABLE_H
+//  Use the standard classes for std::, if available.
+#include <condition_variable>
+
 #include <atomic>
 #include <assert.h>
 #include "mingw.mutex.h"
 #include <chrono>
 #include <system_error>
 #include <windows.h>
-#ifdef _GLIBCXX_HAS_GTHREADS

```

Comment:
I would like to preserve this as a warning

Reply:
See #33.